src/common.c

```diff
@@ -220,18 +234,18 @@ int ofi_check_rx_mode(const struct fi_info *info, uint64_t flags)
 
 uint64_t fi_gettime_ms(void)
 {
-	struct timeval now;
+	struct timespec now;
 
-	gettimeofday(&now, NULL);
-	return now.tv_sec * 1000 + now.tv_usec / 1000;
+	clock_gettime(CLOCK_REALTIME, &now);
```

Comment:
We can check where fi_gettime_ms/us are called, but I'm guessing that CLOCK_MONOTONIC would work.  We likely only want relative times, not the actual time.

Reply:
@shefty Ok, done for relative times.